### PR TITLE
Add some words about the Apache community

### PR DESCRIPTION
**Pull request by NiklasMerz**
I am looking how Apache works from the outside for some time. At the hackathon I got the chance  to talk to some of the Apache commiters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,7 @@ See the different topics that have been worked on:
 * [Singa](achievements/singa.md)
 * [SpamAssassin](achievements/spamassassin.md)
 * [Tomcat](achievements/tomcat.md)
+
+## Further reading
+
+[Observations on the Apache community](theapacheway.md)
```
Comment (jfhovinne) [inline README.md]: Maybe a list here ?

File: theapacheway.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Open Source The Apache Way
+
+The second FOSS Hackathon by EU FOSSA 2 was a good opportinity for outside observers to get a sense of how Open Source works "The Apache Way".
```
Comment (jfhovinne) [inline theapacheway.md]: "opportunity"

File: theapacheway.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Open Source The Apache Way
+
+The second FOSS Hackathon by EU FOSSA 2 was a good opportinity for outside observers to get a sense of how Open Source works "The Apache Way". 
+
+Many Apache Project Management Committee members from different projects were part of the event and showed attendees not part of any Apache community how projects are working at the Apache Software Foundation. Just a week after the Apache Software Foundations 20 year anniversary the Hackathon was an opportunity for some attendees to see how an Open Source community works for the first time. A few members from European institutions got the chance to talk directly to the people involved in the Open Source community which should be a good start for understanding better how to support Open Source projects in the future.
+
+The Apache Way is all about community. Decission are made together and take place in public. It can be hard to understand to understand how and why Open Source and Apache works from the outside. When you start talking to Apache contributors and work with them, even if it is not producing code, you start to get a feeling why it is successful and important.
```
Comment (jfhovinne) [inline theapacheway.md]: "Decisions" are ...
"to understand" (repeated)
"how and why Open Source and Apache work"
"get a feeling of why"

File: theapacheway.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Open Source The Apache Way
+
+The second FOSS Hackathon by EU FOSSA 2 was a good opportinity for outside observers to get a sense of how Open Source works "The Apache Way". 
+
+Many Apache Project Management Committee members from different projects were part of the event and showed attendees not part of any Apache community how projects are working at the Apache Software Foundation. Just a week after the Apache Software Foundations 20 year anniversary the Hackathon was an opportunity for some attendees to see how an Open Source community works for the first time. A few members from European institutions got the chance to talk directly to the people involved in the Open Source community which should be a good start for understanding better how to support Open Source projects in the future.
```
Comment (jfhovinne) [inline theapacheway.md]: Apache Software Foundation

**Review by jfhovinne — CHANGES_REQUESTED** [review]
Thanks @NiklasMerz !
Minor typos.

**Comment by NiklasMerz** [comment]
Thank you for spotting these mistakes. I fixed them and added some words.

**Review by jfhovinne — APPROVED** [review]
Nice work, thanks @NiklasMerz

**Pull request merged**